### ISSUE TITLE
how to train a multi class segment?

### ISSUE THREAD

**Issue by hitxujian** [WAMAWAMA/TNSCUI2020-Seg-Rank1st#3]
thanks for your sharing! if i want to train a multi class segment,how i custom your code

**Comment by WAMAWAMA** [comment]
tks for your issue👍
there are 2 key parts you need to modify (without preprocessing) :
1. you can find two `dataloader` functions in [`/step2to4_train_validate_inference/loader/data_loader.py`](https://github.com/WAMAWAMA/TNSCUI2020-Seg-Rank1st/blob/master/step2to4_train_validate_inference/loader/data_loader.py), which named [`get_loader`](https://github.com/WAMAWAMA/TNSCUI2020-Seg-Rank1st/blob/e6e6a5bb33dbe214409c539aa3bac1af41bb854d/step2to4_train_validate_inference/loader/data_loader.py#L330) and [`get_loader_difficult`](https://github.com/WAMAWAMA/TNSCUI2020-Seg-Rank1st/blob/e6e6a5bb33dbe214409c539aa3bac1af41bb854d/step2to4_train_validate_inference/loader/data_loader.py#L348), and you need to modify the code of the 'processing GT' part in this script (as well as the [`img_mask_aug.py`](https://github.com/WAMAWAMA/TNSCUI2020-Seg-Rank1st/blob/master/step2to4_train_validate_inference/loader/img_mask_aug.py)) to ensure that GT is loaded successfully
2. the loss functions only support binary class segmentation, so you also need to replace the current loss function with a multi-class loss function in the [`solver.py`](https://github.com/WAMAWAMA/TNSCUI2020-Seg-Rank1st/blob/master/step2to4_train_validate_inference/tnscui_utils/solver.py) to ensure that loss is calculated correctly

These changes may not be easy to implement if you are not familiar with my code, so I strongly recommend that you familiarize yourself with the code with the demo dataset (DDTI) before making any changes, and I will consider modifying the code to accommodate multi-class segmentation in the future
